utilities/unittests/test_console.py

```
@@ -3,11 +3,31 @@
 
 
+class SingleAttemptTimeoutSampler:
+    """Run a TimeoutSampler-wrapped function once without retries.
+
+    Patch ``timeout_sampler.TimeoutSampler`` (not ``console.TimeoutSampler``):
+    ``connect()`` uses ``@retry`` from timeout_sampler, which resolves
+    TimeoutSampler from that module at runtime. ``console_eof_sampler`` uses
+    the separate ``console.TimeoutSampler`` import binding.
+    """
+
+    def __init__(self, func, func_args=(), **kwargs):
```

Ruff (0.15.15)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)